pyproject.toml

```diff
@@ -0,0 +1,30 @@
+[tool.ruff]
+line-length = 79
+select = [
+  "E", "F", "W", # flake8
+  "B",           # flake8-bugbear
+  "ARG",         # flake8-unused-arguments
+  "C4",          # flake8-comprehensions
+  "PL",          # pylint
+  "SIM",         # flake8-simplify
+  "T20",         # flake8-print
+  "NPY",         # numpy specific rules
+  "I",           # isort specific rules
+  "UP",          # pyupdate specific rules
+  "C400","C401","C402","C403","C404","C405" # additional pyupgrade rules
+]
+fixable = ["ALL"]
+target-version = "py37"
```

Comment:
py36 is not available in ruff. That is why I checked if 3.6 still works with the current rules. (It does)